Gemfile.lock

```diff
@@ -14,7 +14,7 @@ PATH
       rspec (~> 3.2.0)
 
 GEM
-  remote: https://rubygems.org/
+  remote: http://rubygems.org/
```

Comment:
Then it shouldn't be in this PR.  Even if there is a good reason to include it in this PR, it should be in a separate commit with an explanation in the commit message.